portfolio/src/main/webapp/script.js

```diff
@@ -100,15 +100,15 @@ function scrollToElement(elementId, duration = 300)
  * Then load the comments in that language
  */
 function initComments() {
-  // Make a GET request to "/languages" and parse the response json into "languages" array
+  // Make a GET request to '/languages' and parse the response json into 'languages' array
```

Comment:
Why is it important? Could you extend commit messages with not only 'what' was done, but also 'why' it was done? This will help eliminate comments like this.

Reply:
Thank you for advice! It is important to be consistent with the quotes because it makes the code easier to read and the google code style says to use single quotes, so I decided to use them everywhere.
I will extend my future commits' message to also specify their reason

Reply:
Got it, thanks. In the future, please avoid mixing functional and non-functional (style, quotes, comment typos, etc) changes in one PR. It is always better to move all non-functional changes into a separate PR, and make it clear for a reviewer that these changes do not change functionality. That way it is easier and faster to review and merge.